NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 19.3.6 2020-10-27
+
+* Fixes duplicate service point user constraint violation on tenant upgrade (MODINVSTOR-573)
```

Comment:
Has the maven release been run for this version?

Reply:
Not yet.